mailer.js

```diff
@@ -26,6 +27,31 @@ module.exports = function (log) {
     return 'href="' + url + '" style="color: #0095dd; text-decoration: none; font-family: sans-serif;"'
   }
 
+  function constructLocationString (location) {
+    // construct the location string from the location object
+    var locationString = ''
+    if (location) {
+      if (location.city) {
+        locationString += location.city + ', '
+      }
+      locationString += location.country + ' ' + '(estimated)'
```

Comment:
`'(estimated)'` will also need to be localizable; in the first instance I'm going to assume we can just leave it off of the string here, and include it in the templates like this:

```
{{ location }} {{t (estimated) }}
```